spark-cassandra-connector/src/main/scala/org/apache/spark/sql/cassandra/RddSelectPdTrf.scala

```diff
@@ -0,0 +1,153 @@
+package org.apache.spark.sql.cassandra
+
+import com.datastax.spark.connector.NamedColumnRef
+import com.datastax.spark.connector.cql.TableDef
+import com.datastax.spark.connector.rdd.CassandraRDD
+import org.apache.spark.sql.catalyst.expressions._
+import org.apache.spark.sql.sources.Filter
+import org.apache.spark.sql.sources
+import org.apache.spark.sql.types.DataType
+
+/**
+ * Transform a RDD by filtering [[[Attribute]] and pushdown Catalyst [[Expression]]s
+ */
+class RddPredicateSelectPdTrf(attributes: Seq[Attribute], columnNameByLowercase: Map[String, String],
+                              pushdownPred: Seq[Expression]) extends RddSelectPdTrf {
+
+  override var maybeSelect = if (attributes.nonEmpty) { rdd: RDDType =>
+    rdd.select(attributes.map(a => columnNameByLowercase(a.name): NamedColumnRef): _*)
```

Comment:
can be simplified to `columnNameByLowercase(_)` or use more meaningful name instead of `a`